clients/include/testing_axpy_batched.hpp

```diff
@@ -0,0 +1,266 @@
+/* ************************************************************************
+ * Copyright 2018-2019 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include "cblas_interface.hpp"
+#include "flops.hpp"
+#include "norm.hpp"
+#include "rocblas.hpp"
+#include "rocblas_init.hpp"
+#include "rocblas_math.hpp"
+#include "rocblas_random.hpp"
+#include "rocblas_test.hpp"
+#include "rocblas_vector.hpp"
+#include "unit.hpp"
+#include "utility.hpp"
+
+/* ============================================================================================ */
+template <typename T>
+void testing_axpy_batched_bad_arg(const Arguments& arg)
+{
+    rocblas_local_handle handle;
+
+    rocblas_int N = 100, incx = 1, incy = 1, batch_count = arg.batch_count;
```

Comment:
Not to be addressed in this PR, but we should discuss if want to do auto code review.  e.g.
DCL04-C. Do not declare more than one variable per declaration